microsetta_interface/templates/sitebase.jinja2

```diff
@@ -55,6 +54,9 @@
     <div class="content">
 
     {% if show_breadcrumbs %}
+        <nav class="navbar navbar-expand-lg navbar-light" style="background-color:#e9ecef;">
```

Comment:
These changes with the navbar won't work with the current conditional logic of the template unfortunately. Specifically, the closing `</nav>` on line 70 is not assured to pair with the opening one on line 57. The rendering also looks a little less clean, although I'm not sure if that is due to nested `nav` tags. 

Would it be possible to remove these changes, and we consider them for a separate PR?